util/json.go

```diff
@@ -38,6 +38,11 @@ func UnmarshalJSON(bs []byte, x interface{}) (err error) {
 	return nil
 }
 
+// MarshalJSON parse the JSON encoded data and returns the indented result.
```

Comment:
This comment isn't correct, I think. We're not parsing but serializing here

Reply:
bb61934fc9959e0db8fe76ec53ac7066156c9a49